WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -6788,6 +6792,7 @@
 				CODE_SIGN_ENTITLEMENTS = WordPress.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				COPY_PHASE_STRIP = NO;
+				DEBUG_INFORMATION_FORMAT = dwarf;
```

Comment:
Please please let's remove the `dwarf` optimization, since it's being tested / merged in another PR.

(JUST in case!)